### DISCUSSION TITLE
How to get rid of the sidebar?...

### DISCUSSION THREAD

**Discussion by hitech4444** [aclap-dev/video-downloadhelper#997] [Help and Q&A]
### Your question

Once I activated it from the toolbar I can find no way to disable it.... I really liked it before, when I clicked the VideoDownloader icon... 

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"3664eb7dabad457e"}
platform: x86-32 win
UA: Mozilla/5.0 (Windows NT 6.1; rv:109.0) Gecko/20100101 Firefox/115.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "download_directory": "C:\\Users\\JOHN\\dwhelper",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": true,
        "hide_downloaded": false
    },
    "show_context_menu": true,
    "use_sidebar": true,
    "open_count_store": 24,
    "successfull_dl": 5
}

**Comment by hitech4444** [comment]
Solved it, Sorry...  You must choose "show in pop-up"

**Comment by GrampaWildWilly** [comment]
I have just added a comment to #1192 that could be relevant here.
